jscomp/bin/bsb.ml

```diff
@@ -7346,8 +7416,16 @@ let cwd = Sys.getcwd ()
 let create_bs_config () = 
   ()
 let watch () = 
+  let bsb_watcher =
+    Bsb_build_util.get_bsc_dir cwd // "bsb_watcher.js" in 
+  let bsb_watcher =
+    (*FIXME *)
+    if Sys.win32 then Filename.quote bsb_watcher 
```

Comment:
we need investigate if `Unix.execvp` behaves differently or not between Windows and Unix, (double check)
seems Unix does not need quote while windows does need (when space appears in path)